notebooks/02-ingestion/12-spatial-dimensions.ipynb

```diff
@@ -0,0 +1,709 @@
+    "```\n",
+    "\"filter\": {\n",
+    "    \"type\": \"spatial\",\n",
+    "    \"dimension\": \"address_spatial\",\n",
+    "    \"bound\": {\n",
+    "        \"type\": \"radius\",\n",
+    "        \"coords\": [10.0, 20.0],\n",
+    "        \"radius\": [30.0]\n",
+    "      }\n",
+    "  },\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c6323207-d6c7-4ff5-8b2f-24ed89fe2b86",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "radius_filter_query = {\n",
+    "  \"queryType\": \"topN\",\n",
+    "  \"dataSource\": {\n",
+    "    \"type\": \"table\",\n",
+    "    \"name\": \"example-spatial-index\"\n",
+    "  },\n",
+    "  \"dimension\": {\n",
+    "    \"type\": \"default\",\n",
+    "    \"dimension\": \"marital_status\",\n",
+    "    \"outputName\": \"marital_status\",\n",
+    "    \"outputType\": \"STRING\"\n",
+    "  },\n",
+    "  \"metric\": {\n",
+    "    \"type\": \"dimension\",\n",
+    "    \"ordering\": {\n",
+    "      \"type\": \"lexicographic\"\n",
+    "    }\n",
+    "  },\n",
+    "  \"threshold\": 1001,\n",
+    "  \"intervals\": {\n",
+    "    \"type\": \"intervals\",\n",
+    "    \"intervals\": [\n",
+    "      \"-146136543-09-08T08:23:32.096Z/146140482-04-24T15:36:27.903Z\"\n",
+    "    ]\n",
+    "  },\n",
+    "  \"filter\": {\n",
+    "    \"type\": \"spatial\",\n",
+    "    \"dimension\": \"address_spatial\",\n",
+    "    \"bound\": {\n",
+    "        \"type\": \"radius\",\n",
+    "        \"coords\": [-10.0, 20.0],\n",
+    "        \"radius\": 10.0\n",
+    "      }\n",
+    "  },\n",
+    "  \"granularity\": {\n",
+    "    \"type\": \"all\"\n",
+    "  },\n",
+    "  \"aggregations\": [\n",
+    "    {\n",
+    "      \"type\": \"doubleMin\",\n",
+    "      \"name\": \"min_income\",\n",
+    "      \"fieldName\": \"income\"\n",
+    "    },\n",
+    "    {\n",
+    "      \"type\": \"doubleMax\",\n",
+    "      \"name\": \"max_income\",\n",
+    "      \"fieldName\": \"income\"\n",
+    "    },\n",
+    "    {\n",
+    "      \"type\": \"count\",\n",
+    "      \"name\": \"user_count\"\n",
+    "    }\n",
+    "  ],\n",
+    "  \"context\": {\n",
+    "    \"queryId\": \"2dcbd3c7-cebe-4bb9-8b8b-d1887deda9ea\",\n",
+    "    \"sqlOuterLimit\": 1001,\n",
+    "    \"sqlQueryId\": \"2dcbd3c7-cebe-4bb9-8b8b-d1887deda9ea\",\n",
+    "    \"useNativeQueryExplain\": False\n",
+    "  }\n",
+    "}\n",
+    "\n",
+    "headers = {\n",
+    "  'Content-Type': 'application/json'\n",
+    "}\n",
+    "\n",
+    "result = rest_client.post(\"/druid/v2\", json.dumps(radius_filter_query), headers=headers)\n",
+    "\n",
+    "json.loads(result.text)[0]['result']\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "88ca6f49-1f53-41ce-8ae1-cb89c8f70cf8",
+   "metadata": {},
+   "source": [
+    "### Filter using a polygon area\n",
+    "The next cell is an example of a polygon bound.\n",
```

Comment:
```suggestion
    "The following is an example of a polygon bound.\n",
```

Reply:
Move this line after the polygon definition. Consider combining it with line 555.